### ISSUE TITLE
GUI_i7z compilation errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Hypfer/i7z#7]
```
1. GUI_i7z.cpp:22:17: error: i7z.h: No such file or directory (easily
solved by including "../i7z.h" instead of "i7z.h".

2. GUI_i7z.cpp:238: error: integer constant is too large for ‘long’ type
(I've no idea how so solve that).

3. Warnings:

In file included from GUI_i7z.cpp:22:
i7z.h: In function ‘int turbo_status()’:
i7z.h:113: warning: unused variable ‘i’
i7z.h: In function ‘uint64_t get_msr_value(int, uint32_t, unsigned int,
unsigned int)’:
i7z.h:284: warning: deprecated conversion from string constant to ‘char*’
i7z.h:232: warning: unused variable ‘c’
i7z.h: In function ‘uint64_t set_msr_value(int, uint32_t, uint64_t)’:
i7z.h:297: warning: unused variable ‘c’
i7z.h:298: warning: unused variable ‘pat’
i7z.h:299: warning: unused variable ‘width’
i7z.h:301: warning: unused variable ‘bits’
GUI_i7z.cpp: In constructor ‘MyThread::MyThread()’:
GUI_i7z.cpp:64: warning: comparison between signed and unsigned integer
expressions
GUI_i7z.cpp: In member function ‘virtual void MyThread::run()’:
GUI_i7z.cpp:82: warning: unused variable ‘width’
GUI_i7z.cpp:83: warning: unused variable ‘j’
GUI_i7z.cpp:90: warning: unused variable ‘IA32_MISC_ENABLE’
GUI_i7z.cpp:91: warning: unused variable ‘TURBO_FLAG_low’
GUI_i7z.cpp:92: warning: unused variable ‘TURBO_FLAG_high’
GUI_i7z.cpp:106: warning: unused variable ‘MAX_TURBO_1C’
GUI_i7z.cpp:107: warning: unused variable ‘MAX_TURBO_2C’
GUI_i7z.cpp:108: warning: unused variable ‘MAX_TURBO_3C’
GUI_i7z.cpp:109: warning: unused variable ‘MAX_TURBO_4C’
GUI_i7z.cpp:206: warning: unused variable ‘old_val_C1’
GUI_i7z.cpp:206: warning: unused variable ‘new_val_C1’
GUI_i7z.cpp:210: warning: unused variable ‘tz’
GUI_i7z.cpp: In constructor ‘MyWidget::MyWidget(QWidget*)’:
GUI_i7z.cpp:423: warning: comparison between signed and unsigned integer
expressions
GUI_i7z.cpp:444: warning: comparison between signed and unsigned integer
expressions
GUI_i7z.cpp:471: warning: comparison between signed and unsigned integer
expressions
GUI_i7z.cpp:481: warning: comparison between signed and unsigned integer
expressions
GUI_i7z.cpp:411: warning: unused variable ‘numLogicalCores’
GUI_i7z.cpp:422: warning: unused variable ‘j’
GUI_i7z.cpp: In member function ‘void MyWidget::UpdateWidget()’:
GUI_i7z.cpp:505: warning: comparison between signed and unsigned integer
expressions
GUI_i7z.cpp:511: warning: comparison between signed and unsigned integer
expressions
GUI_i7z.cpp:519: warning: comparison between signed and unsigned integer
expressions

```

Original issue reported on code.google.com by `anestl...@gmail.com` on 21 Feb 2010 at 8:59

**Comment by GoogleCodeExporter** [comment]
```
1. relative path is not the solution. path already added in the Makefile inside 
GUI 
directory. Compiles alright if one uses that Makefile.

2. doesnt happen on my 64-bit machine. 

3. remnant of earlier code which used Intel-CPUID code. Might be adding that 
back in 
future for Nehalem-DP.

Bug-Type: Code prettying 

```

Original comment by `abhirana` on 21 Feb 2010 at 4:53
- Changed state: **Accepted**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
1) - Fixed in the new download 0.21-4
2) - Fixed for 32-bit in the SVN repository. (but its not there yet in the 
download)
3) - Fixed in the new download 0.21-4
```

Original comment by `abhirana` on 18 Mar 2010 at 10:01
- Changed state: **Fixed**
